README.md

```diff
@@ -44,18 +44,37 @@ Depending on the distro you are using will determine which `python3` package to
 | Centos | python3                                                   |
 | Arch   | python                                                    |
 
+### Binary Installation on Linux
 
+This is the fastest way to get started. From the [latest release](https://github.com/AntelopeIO/DUNE/releases/latest) page, download DUNE (mainly python scripts) or visit the [release tags](https://github.com/AntelopeIO/DUNE/releases) page to download specific version of DUNE.
+
+Once you have a `*.deb` file downloaded, you can install it as follows:
+```bash
+sudo apt-get update
+sudo apt-get install -y ~/Downloads/DUNE*.deb
+```
+Your download path may vary.
+
+Finally, verify DUNE was installed correctly in `/usr/opt/DUNE/`. First [add DUNE to Path](#add-dune-to-path) and check:
```

Comment:
Rather than require the user to add `/usr/opt/DUNE` to their path, our package should add a symbolic link from `/usr/bin/dune`.

Reply:
On my ubuntu I already have /usr/bin/dune from some other package installed by default.
When I try to add /usr/bin/dune I get an error that it already exists. So I'm leaving it as it is now.

Reply:
After talking with Scott we decided to install `antelopeio-dune` in `/usr/bin`